bvm/ballerina-core/src/main/java/org/ballerinalang/util/debugger/Debugger.java

```diff
@@ -342,6 +346,44 @@ private MessageDTO generateDebugHitMessage(WorkerExecutionContext ctx, LineNumbe
         FrameDTO frameDTO = new FrameDTO(pck, functionName, callingLine.getFileName(),
                 callingLine.getLineNumber());
         message.addFrame(frameDTO);
+
+        int pkgIndex = ctx.callableUnitInfo.getPackageInfo().pkgIndex;
+
+        PackageVarInfo[] packageVarInfoEntries = ctx.programFile.getPackageInfo(ctx.programFile.getEntryPkgName())
+                .getPackageInfoEntries();
```

Comment:
We generally break new line with period (.) being on the same line and the rest will be moved to next line.

Reply:
Ack. Corrected.